app/main.ipynb

```diff
@@ -56,33 +58,59 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "# write your code here (for Python default syntax)\n",
+    "a = [1,2,3,4,5] * 1000\n",
+    "sys.getsizeof(a)"
```

Comment:
The array `a` should contain zeros, as per the task requirement. Currently, it contains repeated sequences of [1, 2, 3, 4, 5]. Consider initializing `a` with zeros.